sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -346,6 +346,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_tgl_machines[] = {
 		.sof_fw_filename = "sof-tgl.ri",
 		.sof_tplg_filename = "sof-tgl-rt1011-rt5682.tplg",
 	},
+	{
```

Comment:
@plbossart saying the RVP is configured with I2S only and recommending using the soundwire machine driver could throw some people off. Maybe we also need to say that the so_sdw machine driver also supports this config?

Reply:
thanks @ranj063 I updated the wording of the commit message, please let me know if this is clear enough.